packages/data-stores/src/queries/use-site-goals.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+
+export function useSiteGoals( siteId: string | number | undefined ) {
+	return useQuery(
+		'site-goals-' + siteId,
+		async () =>
```

Comment:
Nothing blocking here, just suggestions.

No need to have `async` and `await` here. You can just return the call to `wpcomRequest`. 

However casting types with `as` is not a good practice. Here you could check `if( siteId !== undefined )` before making a call to `wpcomeRequest`.

Reply:
I'll remove the async await, but I don't see a need for checking the `siteId` as the query would be disabled if it's undefined with this:
```
{
			refetchOnWindowFocus: false,
			enabled: !! siteId,         <== this one
		}
```
What do you think @mashikag ?